configuration/device-tree.md

```diff
@@ -332,10 +310,85 @@ Note that a property is assigned the value `false` by not defining it. Boolean p
 name = <&label>,"property?";
 ```
 
-where `label` and `property` are replaced by suitable values. Boolean parameters can cause properties to be created or deleted.
+where `label` and `property` are replaced by suitable values.
+
+Inverted booleans invert the input value before applying it in the same was as a regular boolean; they are declared similarly, but use `!` to indicate the inversion:
```

Comment:
"invert the input value before applying it" - is this just a complicated way of saying they set a false value, or is it more subtle / complicated than that? :confused:

Reply:
It means that if you set the parameter to true the property is set to false (i.e. deleted), while setting it to false makes it true. It's for the case where the sense of the parameter name is opposite to the sense of the property, and also makes radio-button property groups possible.